index.html

```diff
@@ -21,7 +21,17 @@
   <script src="dist/dialog-polyfill.js"></script>
   <link rel="stylesheet" href="dist/dialog-polyfill.css" />
   <style>
-
+button{
+    background-color:#eb427e; ;
```

Comment:
extra `;` at the end should be removed.

Nits:
 - white space consistency
   - `button{` -> `button {`
   - `background-color:#eb427e;` -> `background-color: #eb427e;`
   - indentation of all props should be 2 spaces
   - 0 leading spaces expected before `}`

Reply:
Ok